### PR TITLE
Use GetPropertyChangedSignal for Camera.Changed listeners

### PR DESCRIPTION
**Pull request by DarraghGriffin**
This reduces the amount of overhead when setting busy properties on
Camera such as CFrame in the CameraScripts.

### CODE REVIEW

**Comment by kyle-emmerich** [comment]
Looks good, but we should wait until the flag is going to be removed to merge it. If we got it into the same CI commit or later, perfect. Otherwise GT sites will break. :(

File: CoreScriptsRoot/Modules/VR/Panel3D.lua [hunk 1]
```diff
@@ -1052,24 +1052,22 @@ RunService.Heartbeat:connect(onHeartbeat)
...
-local function onCameraChanged(prop)
-	if prop == "HeadScale" then
-		pcall(function()
-			currentHeadScale = workspace.CurrentCamera.HeadScale
-		end)
-		for i, v in pairs(panels) do
-			v:OnHeadScaleChanged(currentHeadScale)
-		end
+local function onHeadScalePropChanged(prop)
+	pcall(function()
```
Comment (kyle-emmerich) [inline CoreScriptsRoot/Modules/VR/Panel3D.lua]: No big deal, but we can probably remove this pcall now.

**Comment by hockeychaos** [comment]
we good to merge?

**Comment by DarraghGriffin** [comment]
Going to commit this with the new system.

**Pull request closed without merging**